requirements.txt

```diff
@@ -28,6 +28,6 @@ pyyaml==3.*
 vulture==0.10.*
 bandit==1.1.*
 nbformat>=4.*
-pyflakes==1.2.*  # Although we don't need this directly, solves a dep conflict
+pyflakes==1.2.*
```

Comment:
The requirement for flake8 seems to be a version < 1.3.0 -> [test failure](https://circleci.com/gh/coala/coala-bears/3774?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)